docs/source/quick_start.rst

```diff
@@ -0,0 +1,199 @@
+============
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+-----------------------------
+Download Latest, Set Up Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" will need to be updated for each release 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/tag/4.2.01>`
+
+
+::
+
+  unzip kokkos-x.y.z.zip (or tar -xzf kokkos-x.y.z.tar.gz)
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+-------------------
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
```

Comment:
```suggestion
You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
```

Reply:
For the `note` inset to work, you must indent.

Reply:
I think here it makes more sense to have it prose rather than a note